### ISSUE TITLE
Percentage not visible on last segment of the chart sometimes

### ISSUE THREAD

**Issue by shripalsoni04** [emn178/Chart.PieceLabel.js#3]
First of all, 👍  for this very helpful plugin.

I found that sometimes the percentage on chart do not show in last segment of the doughnut chart. I tried to debug the issue and found that issue only comes when percentage total go above 100 while adding percentage of last segment. The issue is at below line

```
percentage -= totalPercentage - 100;
```

The problem is, once we deduct the overshoot percentage, it will not take the precision into consideration and will result in very long number after decimal point. Due to that it is not getting drawn on last segment.

I think, the solution  is to trim the numbers again using `toFixed` as shown below
```
percentage = parseFloat(percentage.toFixed(precision));
```

I did above change in forked repo https://github.com/shripalsoni04/Chart.PieceLabel.js and it is working as expected now.

If the above solution is proper, please release a patch with it or you can create a PR from https://github.com/shripalsoni04/Chart.PieceLabel.js and merge it.

Let me know in case of any concerns.

Thanks.

**Comment by emn178** [comment]
please pull request, I'll merge this. thanks.

**Comment by emn178** [comment]
this fix has included in v0.3.0. thanks.
